index.js

```diff
@@ -0,0 +1,140 @@
+const ListURL = 'https://trektravel.herokuapp.com/trips';
+const DetailURL = 'https://trektravel.herokuapp.com/trips';
+
+const reserveHTML = $(`<h1>Reserve a spot on the Trip</h1>
+<form id="reserve-form">
+  <div>
+    <label for="name">Name</label>
+    <input type="text" name="name" />
+  </div>
+
+  <div>
+    <label for="email">Email</label>
+    <input type="text" name="email" />
+  </div>
+
+  <input type="submit" name="reserve" value="Reserve spot" />
+</form>`);
+
+const displayStatus = (message) => {
+    $('#status-message').html(message);
+};
+const handleApiError = (message, errors) => {
+    let content = `<p>${message}</p><ul>`;
+    for (const field in errors) {
+        for (const problem of errors[field]) {
+            content += `<li>${field}: ${problem}</li>`;
+        }
+    }
+    content += "</ul>";
+    displayStatus(content);
+};
+
+const loadTrips = () => {
+    displayStatus("loading trips...");
+
+    const tripList = $('#trip-list');
+    tripList.empty();
+    const tripReserve = $('.reserve-trip')
+
+    axios.get(ListURL)
+
+        .then((response) => {
+            displayStatus(`Successfully loaded ${response.data.length} trips`);
+            response.data.forEach((trip) => {
+                const tripHTML = $(`<li><a href="#">${trip.name}</a></li>`);
+                tripList.append(tripHTML);
+
+                tripHTML.click(() => {
+                    showTripDetails(trip.id);
+                    tripReserve.append(reserveHTML);
+                    $('#reserve-form').submit(() => {
+                        reserveTrip(trip.id);
```

Comment:
When reserving a second trip, your site makes two POST requests. To reproduce:
1. Open page
1. Open chrome dev tools to the "Network" tab
1. Click See All Trips
1. Click on the first trip (Cairo to Zanzibar)
1. Make a reservation and submit
1. Click on the second trip (Everest Base Camp Trek)
1. Make a reservation and submit
1. Observe that on submit, 2 POST requests were made to trek API's /reservations endpoint

The reason for this is that you reuse the same reservation form for each trip, but re-attach the `submit` handler every time you show details for a trip.